lib/file_database.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'json'
+
+class FileDatabase
+  def initialize(file_dir)
+    @file = File.open(file_dir, 'r')
+  end
+
+  def all
+    file.rewind
+    convert_json(file.read)
+  end
+
+  def database_empty?
+    all.any?
+  end
+
+  private
+
+  def convert_json(contacts)
+    return [] if contacts.empty?
+
+    JSON.parse(contacts, { symbolize_names: true })
+  end
+
+  attr_reader :file
```

Comment:
usually the instance attributes go right below the `private` rather than at the end of the class